projects/cmake/build.sh

```diff
@@ -160,10 +158,6 @@ if [ "$static_boost" = "true" ] ; then
     cmake_args="-DSTATIC_BOOST=ON $cmake_args"
 fi
 
-if [ "$help2yml" = "true" ] ; then
-    cmake_args="-DHELP=ON $cmake_args"
```

Comment:
Deleting this line means that we will NOT build rb-help2yml, right?

Reply:
My mistake. Will revert it